src/appshell/qml/Preferences/EditPreferencesPage.qml

```diff
@@ -22,16 +25,45 @@ PreferencesPage {
         width: parent.width
         spacing: root.sectionsSpacing
 
+        EffectBehaviorSection {
+            editPreferencesModel: editPreferencesModel
+
+            navigation.section: root.navigationSection
+            navigation.order: root.navigationOrderStart
+        }
+
+        SeparatorLine { }
+
+        AsymmetricStereoHeightsSection {
+            id: asymmetricStereoHeightsSection
+            editPreferencesModel: editPreferencesModel
+
+            navigation.section: root.navigationSection
+            navigation.order: root.navigationOrderStart + 1
+        }
+
+        SeparatorLine { }
+
+        PastingBehaviorSection {
+            editPreferencesModel: editPreferencesModel
+
+            navigation.section: root.navigationSection
+            navigation.order: asymmetricStereoHeightsSection.navigationOrderEnd + 1
+        }
+
+        SeparatorLine { }
+
         MonoStereoConversionSection {
             id: monoStereoConversionSection
+
             askBeforeConverting: editPreferencesModel.askBeforeConvertingToMonoOrStereo
+
             navigation.section: root.navigationSection
-            navigation.order: root.navigationOrderStart
+            navigation.order: root.navigationOrderStart + 4
```

Comment:
A proposal that will simplify the solution and we will not have to think about what number to write here and will also work if we decide to add another control to the section above:
It is better to use a chain for navigation - each next section uses the navigation of the previous section.

You can add navigationOrderEnd to BaseSection and reuse it wherever necessary